src/py21cmfast/src/ps.c

```diff
@@ -1529,6 +1537,51 @@ void free_lvector(unsigned long *v, long nl, long nh)
     free((FREE_ARG) (v+nl-NR_END));
 }
 
+
+/* dnbiasdM */
+double dnbiasdM(double M, float z, double M_o, float del_o){
+    double sigsq, del, sig_one, sig_o;
+
+//    if ((M_o-M) < TINY){
+//        fprintf(stderr, "WARNING:  In function dnbiasdM: M must be less than M_o!\nAborting...\n");
+//        return -1;
+//    }
+    if ((M_o-M) < TINY){
+        LOG_ERROR(" WARNING:  In function dnbiasdM: M must be less than M_o!\nAborting...\n");
```

Comment:
Should just be `LOG_ERROR("M must be less than M_o! Got M=%g, M_o=%g", M, M_o")` then Throw a ValueError.